install.sh

```diff
@@ -102,15 +109,17 @@ fi
 
 if [[ "$ARCH" == "x86_64" ]]; then
   LIB_SUFFIX='64'
+else
+  LIB_SUFFIX=''
```

Comment:
Is LIB_SUFFIX not already empty by default, since we don't set the variable?

Reply:
It doesn't hurt to set it explicitly for clarity and in the off chance it's set outside of crew.

Reply:
Reduce?
```
LIB_SUFFIX=''
[[ "${ARCH}" == "x86_64" ]] && LIB_SUFFIX='64'
```